packages/field-plugin/src/utils/isSerializable/isSerializable.ts

```diff
@@ -0,0 +1,8 @@
+export const isSerializable = (obj: unknown): boolean => {
+  try {
+    JSON.parse(JSON.stringify(obj))
```

Comment:
Are you sure about this implementation? It's not only about JSON-stringifiable, but also we need to check if it can be sent via postMessage.

https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage

Reply:
To add more info, your implementation just checks if it throws or not while serializing and deserializing, but according to [the MDN docs](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Structured_clone_algorithm), 

![Screenshot 2023-08-14 at 16 08 19](https://github.com/storyblok/field-plugin/assets/499898/17558f48-2f1a-4487-bbee-0283305be70b)

You cannot pass `Function` via `postMessage`. However this implementation returns `true` for functions too.